src/com/tictacgo/TicTacGoMenu.java

```diff
@@ -0,0 +1,40 @@
+package com.tictacgo;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.view.View.OnClickListener;
+import android.widget.RadioGroup;
+import android.widget.TextView;
+
+
+public class TicTacGoMenu extends Activity {
+	
+	/** Called when the activity is first created. */
+  @Override
+  public void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    setContentView(R.layout.menu);
+
+    findViewById(R.id.playButton).setOnClickListener(new OnClickListener() {
+      public void onClick(View v) {
+        Intent playGame = new Intent(v.getContext(), TicTacGoGame.class);
+
+        String p1Name = LayoutUtils.getTextOrHint((TextView) findViewById(R.id.localPlayerOneName));
+        playGame.putExtra("p1Name", p1Name);
```

Comment:
This feels sloppy. Let's do 2 things:
1) Pull out "p1Name", etc. into constants. Preferably, prefix them with "com.tictacgo" to prevent name collisions with intents from other apps.
2) Pass the starting Player as an extra instead of a resource ID (line 31).